### DISCUSSION TITLE
Difference between "Image Tab" and "Move Images to Other Tab"?

### DISCUSSION THREAD

**Discussion by waseemv** [hluk/CopyQ#2491] [Q&A]
On the CopyQ Commands [page](https://github.com/hluk/copyq-commands/tree/master/Automatic), there are two different commands that seem to do the same thing (i.e. save images in a dedicated tab):

**Image Tab**
**Move Images to Other Tab**

Can someone please mention the differences between the two?

**Comment by GFDGIT** [comment]
They are almost exactly the same thing.

**Comment by hluk** [comment]
I think I added "Move Images to Other Tab" not realizing there is already a command that does this. I will remove the new command.
